src/components/bookItem.js

```diff
@@ -39,14 +43,29 @@ const Books = (props) => {
           </button>
         </div>
       </div>
+      <div className="status">
+        <img src={spinner} alt="progression circle" className="spinner-img" />
+        <div>
+          <p className="percentage">0%</p>
+          <p className="status-details">Incomplete</p>
+        </div>
+      </div>
+      <section className="position">
+        <h2 className="pos-header">CURRENT CHAPTER</h2>
+        <p className="pos-details">Chapter 1</p>
```

Comment:
- [ ] also this looks darker in your design than it is in zeplin design, kindly fix this